docs/source/release-notes.rst

```diff
@@ -11,6 +11,8 @@ v1.7 Notes
 v1.7.2 (2022-06-07)
 -------------------
 - Improved documentation and examples.
+- Improved ``to_pandapower`` performance with vectorized conversion.
```

Comment:
These should go to a new version since 1.7.2 is already out.

Add a section new section above and move the two to the new section.

Reply:
Thanks for your instruction!